<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sv4gui_MachineLearning</class>
 <widget class="QWidget" name="sv4gui_MachineLearning">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>759</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QToolBox" name="toolBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="segPage">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>501</width>
           <height>692</height>
          </rect>
         </property>
         <attribute name="label">
          <string>ML 2D Segmentation</string>
         </attribute>
         <widget class="QPushButton" name="msgButton_2">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>400</y>
            <width>131</width>
            <height>27</height>
           </rect>
          </property>
          <property name="text">
           <string>Segment Paths</string>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>30</y>
            <width>67</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Paths</string>
          </property>
         </widget>
         <widget class="QListWidget" name="pathList">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>70</y>
            <width>256</width>
            <height>192</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="intervalEdit">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>280</y>
            <width>113</width>
            <height>27</height>
           </rect>
          </property>
          <property name="text">
           <string>10</string>
          </property>
         </widget>
         <widget class="QLabel" name="intervalLabel">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>280</y>
            <width>67</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Interval</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="fourierEdit">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>310</y>
            <width>113</width>
            <height>27</height>
           </rect>
          </property>
          <property name="text">
           <string>7</string>
          </property>
         </widget>
         <widget class="QLabel" name="fourierLabel">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>310</y>
            <width>101</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Fourier Modes</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="selectAllPathsCheckBox">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>50</y>
            <width>131</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>select all paths</string>
          </property>
         </widget>
         <widget class="QPushButton" name="sampleNetButton">
          <property name="geometry">
           <rect>
            <x>300</x>
            <y>10</y>
            <width>171</width>
            <height>27</height>
           </rect>
          </property>
          <property name="text">
           <string>Sample New Network</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
